cypress/integration/pages/photoGalleryPage/testsForAMPOnly.js

```diff
@@ -0,0 +1,37 @@
+import config from '../../../support/config/services';
+// import envConfig from '../../../support/config/envs';
+// import appConfig from '../../../../src/server/utilities/serviceConfigs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForAMPOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForAMPOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForAMPOnly = ({
+  service,
+  pageType,
+}) =>
+  describe(`testsThatFollowSmokeTestConfigForAMPOnly for ${service} ${pageType}`, () => {
+    describe('AMP Status', () => {
+      it('should return a 200 response', () => {
```

Comment:
Hmm I can see we're following a pattern here, but I'm curious why we don't do this in testsForAllPages. Do you happen to know why?

Reply:
I asked @sareh about this specific test because I wondered the same thing and this test should be placed in `testsForAllAmpPages`. But It'll have to be excluded for 404 error pages and article page. 

Wouldn't suppose you would know why this test isn't ran on article pages?

Reply:
Having done a bit of digging, I found this on line 22 of runTestsForPage. Each page visited by Cypress using this function is tested for a 200. 

```js
const visitPage = (path, pageType) => {
  const expectedContentType = 'text/html';
  const isErrorPage = pageType.includes('error');
  const expectedStatus = isErrorPage ? 404 : 200;
  const failOnStatusCode = !isErrorPage;

  cy.testResponseCodeAndType(path, expectedStatus, expectedContentType);
  cy.visit(path, {
    failOnStatusCode,
  });
};
```

This needs verifying, but we appear to be manually duplicating this logic in the following files:
- frontPage AMP only
- liveRadio AMP only
- mediaAsset AMP only
- _photoGallery AMP only_

Reply:
Could we resolve this duplication in a follow-up PR?